openapi-contracts/src/test/java/com/sinch/sdk/domains/verification/models/dto/v1/start/StartVerificationResponseTest.java

```diff
@@ -0,0 +1,111 @@
+
+  public static StartVerificationResponseInternal expectedStartVerificationFlashCallDto =
+      new StartVerificationResponseInternalImpl(
+          (StartVerificationResponseFlashCallImpl)
+              StartVerificationResponseFlashCall.builder()
+                  .setId("the id")
+                  .setMethod(VerificationMethod.FLASHCALL.value())
+                  .setLinks(LinkTest.expectedLinks)
+                  .setFlashCall(
+                      StartVerificationResponseFlashCallContent.builder()
+                          .setCliFilter("(.*)5312(.*)")
+                          .setInterceptionTimeout(45)
+                          .setReportTimeout(75)
+                          .setDenyCallAfter(0)
+                          .build())
+                  .build());
+
+  @GivenJsonResource("/domains/verification/v1/start/StartVerificationDataResponseDto.json")
+  StartVerificationResponseInternal loadedStartVerificationDataDto;
+
+  public static StartVerificationResponseInternal expectedStartVerificationDataDto =
+      new StartVerificationResponseInternalImpl(
+          (StartVerificationResponseDataImpl)
+              StartVerificationResponseData.builder()
+                  .setId("the id")
+                  .setMethod(VerificationMethod.SEAMLESS.value())
+                  .setLinks(LinkTest.expectedLinks)
+                  .setSeamless(
+                      StartVerificationResponseDataContent.builder()
+                          .setTargetUri("target URI")
+                          .build())
+                  .build());
+
+  @GivenJsonResource("/domains/verification/v1/start/StartVerificationSmsResponseDto.json")
+  StartVerificationResponseInternal loadedStartVerificationSmsDto;
+
+  public static StartVerificationResponseInternal expectedStartVerificationSmsDto =
+      new StartVerificationResponseInternalImpl(
+          (StartVerificationResponseSmsImpl)
+              StartVerificationResponseSms.builder()
+                  .setId("the id")
+                  .setMethod(VerificationMethod.SMS.value())
+                  .setLinks(LinkTest.expectedLinks)
+                  .setSms(
+                      StartVerificationResponseSmsContent.builder()
+                          .setTemplate("Your verification code is {{CODE}}. Verified by Sinch")
+                          .setInterceptionTimeout(298)
+                          .build())
+                  .build());
+
+  @Test
+  void deserializeStartPhoneCall() {
+    TestHelpers.recursiveEquals(
+        loadedStartVerificationPhoneCallDto, expectedStartVerificationPhoneCallDto);
+  }
+
+  @Test
+  void deserializeStartFlashCall() {
+    TestHelpers.recursiveEquals(
+        loadedStartVerificationFlashCallDto, expectedStartVerificationFlashCallDto);
+  }
+
+  @Test
+  void deserializeStartData() {
+    TestHelpers.recursiveEquals(loadedStartVerificationDataDto, expectedStartVerificationDataDto);
+  }
+
+  @Test
+  void deserializeSms() {
```

Comment:
Nit: should be renamed `deserializeStartSms`

Reply:
Done